src/manifest.c

```diff
@@ -675,19 +675,21 @@ char *file_type_to_string(struct file *file)
 	return type;
 }
 
+/* Calculate the contentsize for the manifest based on file sizes.
+ * This should calculate the files uniquely included in this manifest, but none
+ * of its submanifests, which will allow calculation of all bundles on a system
+ * by adding all manifest->contentsizes of installed bundles.
+ */
```

Comment:
When a set of bundles are part of the same include hierarchy, the description of how clients can calculate total bundle size is true, but the description is not true in general.

For example, consider two identical bundles, A and B. Both bundles include bundle C. The contentsize of bundle C is subtracted from the size of bundles A and B to produce contentsize for A and B. However, if a client installs both bundles A and B, summing their contentsize values will be an overestimate, because they overlap 100%.

Can you clarify the doc string?